integrations/oci/objectstorage/src/main/java/io/helidon/integrations/oci/objectstorage/OciObjectStorageRxImpl.java

```diff
@@ -84,6 +85,18 @@
         return restApi.post(apiPath, request, RenameObject.Response.builder());
     }
 
+    @Override
+    public Single<ApiOptionalResponse<GetObjectRx.Response>> getBucket(GetObject.Request request) {
```

Comment:
getBucket should probably not have `GetObject.Request` request, as that is for a streaming response. Bucket is probably a different type. If you want to have `getBucket`, please create a new type `GetBucket` with `Request` and `Response` inner classes that have proper fields.

Reply:
see https://docs.oracle.com/en-us/iaas/api/#/en/objectstorage/20160918/Bucket/GetBucket